Correct steps for linux installation

alvinsiew opened the pull request.

micahsnyder commented:
It looks like the same issue exists for:
https://github.com/Cisco-Talos/clamav-faq/blob/master/manual/UserManual/Installation-Unix/Steps-macOS.md
https://github.com/Cisco-Talos/clamav-faq/blob/master/manual/UserManual/Installation-Unix/Steps-Debian-Ubuntu.md

Can I trouble you to update it in these other 2 locations as well?

alvinsiew commented:
Sure! Updated! Corrected his as well  "sudo yum install valgrind check
check-devel".
Check-devel is require.

On Fri, Dec 6, 2019 at 3:34 AM Micah Snyder <notifications@github.com>
wrote:

> It looks like the same issue exists for:
>
> https://github.com/Cisco-Talos/clamav-faq/blob/master/manual/UserManual/Installation-Unix/Steps-macOS.md
>
> https://github.com/Cisco-Talos/clamav-faq/blob/master/manual/UserManual/Installation-Unix/Steps-Debian-Ubuntu.md
>
> Can I trouble you to update it in these other 2 locations as well?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Cisco-Talos/clamav-faq/pull/123?email_source=notifications&email_token=AJLZGFJOYV65WAVD6CXIIG3QXFJVRA5CNFSM4JVCKKH2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEGB34BY#issuecomment-562281991>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJLZGFOBOUKVNKFYEJR6JKTQXFJVRANCNFSM4JVCKKHQ>
> .
>
